src/components/richText/imageNode/LexicalImage.tsx

```diff
@@ -0,0 +1,36 @@
+import type { LexicalEditor, NodeKey } from "lexical";
+
+import * as React from "react";
+import { useRef, useMemo } from "react";
+
+interface LexicalImageProps {
+  src: string;
+  altText: string;
+  maxWidth: number;
+  width: "inherit" | number;
+  height: "inherit" | number;
+  nodeKey: NodeKey;
+  showCaption: boolean;
+  captionsEnabled: boolean;
+  caption: LexicalEditor;
+}
+
+export default function LexicalImage({
+  src,
+  altText,
+  width,
+  height,
+  maxWidth,
+}: LexicalImageProps): JSX.Element {
+  const imageRef = useRef<null | HTMLImageElement>(null);
```

Comment:
why is this ref needed? is to support later functionality?

Reply:
Yes, it's needed to support later functionality.